app/models/grda_warehouse/auth_policies/project_policy.rb

```diff
@@ -8,7 +8,7 @@ class GrdaWarehouse::AuthPolicies::ProjectPolicy < GrdaWarehouse::AuthPolicies::
   # expose role permissions. Optionally rename the permission
   [
     [:can_edit_projects, :can_edit?],
-    [:can_delete_projects, :can_delete?],
+    [:can_delete_projects, :can_destroy?],
```

Comment:
bug fix, standardize on `can_destroy` which seems more intuitive to me; perhaps that's subjective